config/prow/config.yaml

```diff
@@ -66,6 +66,18 @@ prowjob_namespace: default
 pod_namespace: test-pods
 log_level: debug
 
+slack_reporter:
+  job_types_to_report:
+    - postsubmit
+    - periodic
+    - batch
```

Comment:
Batches are a form of presubmit and should probably be excluded just like `presubmit` was since we don't expect them to always pass.